saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/detail/JudgmentDetailController.java

```diff
@@ -7,6 +7,7 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 
 import pl.edu.icm.saos.persistence.repository.JudgmentRepository;
+import pl.edu.icm.saos.webapp.judgment.detail.correction.JudgmentCorrectionService;
```

Comment:
Łukasz, think about changing the name JudgmentDetailController/JudgmentDetailControllerTest into JudgmentDetailsController/JudgmentDetailsControllerTest (the same situation as with the method name).